torchmetrics/image/fid.py

```diff
@@ -148,10 +148,6 @@ class FrechetInceptionDistance(Metric):
         is installed. Either install as ``pip install torchmetrics[image]`` or
         ``pip install torch-fidelity``
 
-    .. note:: the ``forward`` method can be used but ``compute_on_step`` is disabled by default (oppesit of
-        all other metrics) as this metric does not really make sense to calculate on a single batch. This
-        means that by default ``forward`` will just call ``update`` underneat.
```

Comment:
I would keep it, just remove mention of ``compute_on_step``
cc: @SkafteNicki

Reply:
No, if you remove the mention it becomes meaningless IMO. I'd delete it

Reply:
yes delete it